flipperdevices/Flipper-Android-App
# Migrate preference module to kmp

**Background**

Right now, :core:preference is the bottle neck for migrating an entire project to KMP. With this PR we are migrating :core:preference to KMP structure

**Changes**

- Use wire instead of protoc
- Migrate :core:preference to KMP
- Add `anvil-multiplatform` gradle plugin

**Test plan**

- Open app from `dev` branch or release version
- Change some settings
- Launch app from this branch and check that all settings remain the same
